cmd/prometheus/main.go

```diff
@@ -1174,7 +1172,8 @@ func main() {
 						}
 						currentChecksum, err := config.GenerateChecksum(cfg.configFile)
 						if err != nil {
-							logger.Error("Failed to generate checksum during configuration reload", "err", err)
+							checksum = currentChecksum
+							logger.Error("msg", "Failed to generate checksum during configuration reload", "err", err)
```

Comment:
This should take care of the linting failure:
```suggestion
							logger.Error("Failed to generate checksum during configuration reload", "err", err)
```